internal/cucumber/config.go

```diff
@@ -193,6 +198,13 @@ func Validate(p *Project) error {
 		}
 
 		p.Suites[i].Options.Paths = fpath.ExcludeFiles(files, excludedFiles)
+
+		if p.Sauce.Retries > 0 && v.PassThreshold > p.Sauce.Retries {
+			return fmt.Errorf(msg.InvalidPassThreshold)
+		}
```

Comment:
Hm. Not sure that's enough 🤔 Consider the case where the user sets the `passThreshold` to _4_, but doesn't set any `retries`. Might mislead the user to believing that `passThreshold` is being respected.

I think the math at all times is:
```
retries >= passThreshold - 1
```

Examples:
- Retry 0..n; Threshold 1 --> OK (since there's at least 1 attempt and pass-threshold is also 1)
- Retry 1; Threshold 3 --> NOK (since there'll be only 2 attempts and pass-threshold is 3)

Reply:
make sense!